### PR TITLE
Replaced ActionableMessage In send-content

### PR DESCRIPTION
**Pull request by pritam1813**
## Explanation
- This PR fixes part of #19528
- Replaces instances of `ActionableMessage` with `BannerAlert` in `ui/pages/send/send-content/send-content.component.js`

### Before
before

### After
after

https://github.com/MetaMask/metamask-extension/assets/8112138/eb33fd46-8bf0-410e-83c5-601355c418df

## Manual Testing Steps
- Create and Install New Build from this branch
- After importing/creating account, click send
- Enter any valid contract address in the `public address` field to trigger the component.

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/pages/send/send-content/send-content.component.js [hunk 1]
```diff
@@ -103,15 +104,9 @@ export default class SendContent extends Component {
...
-          iconFillColor="var(--color-error-default)"
-          primaryActionV2={{
-            label: t('tooltipApproveButton'),
-            onClick: acknowledgeRecipientWarning,
-          }}
-          message={t('sendingToTokenContractWarning', [
+        <BannerAlert
+          severity={Severity.Danger}
+          description={t('sendingToTokenContractWarning', [
             <a
```
Comment (georgewrmarshall) [inline ui/pages/send/send-content/send-content.component.js:110]: I think we could use `ButtonLink` here(Will need to import `ButtonLinkSize` from component-library
```suggestion
            <ButtonLink
              key="contractWarningSupport"
              href={CONTRACT_ADDRESS_LINK}
              externalLink
              size={ButtonLinkSize.Inherit}
            >
              {t('learnMoreUpperCase')}
            </ButtonLink>,
```

We could also remove the styles for  className="send__warning-container__link" https://github.com/MetaMask/metamask-extension/blob/db064780d814477e71e3167ae34970e23ded09be/ui/pages/send/send.scss#L41-L43

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Looking good @pritam1813! Made one suggestion

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Looking good! Left one suggestion

File: ui/pages/send/send-content/send-content.component.js [hunk 2]
```diff
@@ -2,7 +2,8 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import PageContainerContent from '../../../components/ui/page-container/page-container-content.component';
 import Dialog from '../../../components/ui/dialog';
-import ActionableMessage from '../../../components/ui/actionable-message';
+import { BannerAlert } from '../../../components/component-library';
```
Comment (georgewrmarshall) [inline ui/pages/send/send-content/send-content.component.js:5]: Need to import `ButtonLink` as well
```suggestion
import { BannerAlert, ButtonLink } from '../../../components/component-library';
```

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Hey @pritam1813, I think you are missing an import. I have commented.

File: ui/pages/send/send-content/send-content.component.js (hunk 2 above)
Comment (pritam1813) [reply ui/pages/send/send-content/send-content.component.js:5]: Sorry, I overlooked earlier and committed your suggestions directly.

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! I updated the e2e tests so they will hopefully pass now. Thanks for your contribution @pritam1813

**Pull request closed without merging**